service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ImmunizationObservationStatementMapperTest.java

```diff
@@ -51,6 +52,10 @@
         + "immunization-reason-not-given-text.json";
     private static final String INPUT_JSON_WITH_VACCINE_CODE = IMMUNIZATION_FILE_LOCATIONS
         + "immunization-vaccine-code-given.json";
+    private static final String INPUT_JSON_WITHOUT_PRACTITIONER = IMMUNIZATION_FILE_LOCATIONS
+        + "immunization-no-practitioner.json";
+    private static final String INPUT_JSON_WITHOUT_PRACTITIONER_ACTOR = IMMUNIZATION_FILE_LOCATIONS
```

Comment:
Think this should be renamed `INPUT_JSON_WITH_PRACTITIONER_BUT_NO_ACTOR`

Reply:
Sure